test/groovy/util/FixtureHelper.groovy

```diff
@@ -715,11 +716,490 @@ class FixtureHelper {
         )
     }
 
+    static Map createSockShopTestResults() {
+        return JUnitParser.parseJUnitXML(
+            createSockShopJUnitXmlTestResults()
+        )
+    }
+
     static Set createTestResultErrors() {
         return JUnitParser.Helper.getErrors(createTestResults())
     }
 
+    static Set createSockShopTestResultErrors() {
+        return JUnitParser.Helper.getErrors(createSockShopTestResults())
+    }
+
     static Set createTestResultFailures() {
         return JUnitParser.Helper.getFailures(createTestResults())
     }
+
+    static Set createSockShopTestResultFailures() {
+        return JUnitParser.Helper.getFailures(createSockShopTestResults())
+    }
+
+    static List createIssuesForJQLQuery() {
+        return [
+            [
+                key   : "TESTCAL-20",
+                fields: [
+                    summary    : "DevOps Epic for Test",
+                    description: "Some issue descripion",
+                    status     : [
+                        name: "Open"
+                    ],
+                    labels     : ["LeVA_Doc:CSD"]
+                ]
+            ]
+        ]
+    }
+
+    static Map createProjectVersion() {
+        return [
+            "id"  : "11100",
+            "name": "0.3"
+        ]
+    }
+
+    static List createSockShopJiraTestIssues(){
+        return new JsonSlurperClassic().parseText(
+"""
```

Comment:
@georgfedermann all sock shop test data needs to come from the central .json file.